src/components/MovieDetail.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import React from 'react';
+import { FetchData } from '@/lib/types';
+import Image from 'next/image';
+
+type MovieDetailProps = {
+  movie: FetchData['results'][0];
+};
+
+export default function MovieDetail({ movie }: MovieDetailProps) {
+  if (!movie) {
+    return <div className="text-white">Loading...</div>;
+  }
+
+  return (
+    <div className="text-white">
+      <div className="relative w-full h-[415px] overflow-hidden mb-4">
+        <Image
+          src={`https://image.tmdb.org/t/p/w500${movie.poster_path}`}
```

Comment:
```suggestion
          src={`https://image.tmdb.org/t/p/w1280${movie.poster_path}`}
```

약간 렌더링 되는 이미지의 해상도가 살짝 아쉬운데, 위와 같이 너비를 다르게 해서 요청을 날려보시면 어떨까 싶어요!